src/main/java/sirius/biz/storage/layer2/URLBuilder.java

```diff
@@ -609,7 +610,7 @@ private String computeAccessToken(String authToken) {
 
     private void appendAddonText(StringBuilder result) {
         if (Strings.isFilled(addonText)) {
-            result.append(Strings.reduceCharacters(NON_URL_CHARACTERS.matcher(addonText).replaceAll("-")));
+            result.append(StringCleanup.reduceCharacters(NON_URL_CHARACTERS.matcher(addonText).replaceAll("-")));
```

Comment:
This should better match the intended cleanup call structure.
Please check the suggestion for validity in the IDE before committing.
```suggestion
            result.append(Strings.cleanup(addonText, text -> NON_URL_CHARACTERS.matcher(addonText).replaceAll("-"), StringCleanup::reduceCharacters));
```

Reply:
This is 1 to 1 was the replaced deprecated method calls internally 😅 Why is it even deprecated when its semantics changed or am I missing something??

Reply:
Note the class JavaDoc in [`StringCleanup`](https://github.com/scireum/sirius-kernel/blob/0a9e8b37f2a9ce68a3cbb8526d717994b3ed9b90/src/main/java/sirius/kernel/commons/StringCleanup.java#L23):
```
/**
 * Provides various methods to clean-up or reduce strings.
 * <p>
 * Please note, that most of the functions provided here are intended to be used via {@code Strings.cleanup(..)} and
 * therefore provide no internal <tt>null</tt> check.
...

Reply:
`@deprecated Use {@link StringCleanup#reduceCharacters(String)} or {@code Strings.cleanup(input, Cleanup::reduceCharacters)} instead`